test/integration/restart_task_test.rb

```diff
@@ -198,44 +199,6 @@ def test_restart_not_existing_namespace
       in_order: true)
   end
 
-  def test_restart_failure
```

Comment:
I'm not sure how to go about forcing a failure in this case, since we are no longer pushing an env variable into a container (and I don't really see the need to do so, anyway).

Reply:
I think you could use the [downward api](https://kubernetes.io/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information/#store-pod-fields). In that example they mount pod annotations into the local file system.

A more convoluted alternative would be to kubectl inside the pod and read the annotations off the well known deployment.